dss-server/src/main/java/com/webank/wedatasphere/dss/server/entity/DWSFavorite.java

```diff
@@ -0,0 +1,57 @@
+package com.webank.wedatasphere.dss.server.entity;
+
+/**
+ * Created by Adamyuanyuan on 2020/6/25
+ */
+public class DWSFavorite extends BaseEntity{
+
```

Comment:
DWS is a pretty old abbreviation of DataSphere Studio, and will be removed step by step.
Can we replace DWS to DSS? Thanks.

Reply:
Thanks for the reminder, I will solve it in next PR.